demo/runners/faber.py

```diff
@@ -36,7 +36,7 @@ def __init__(
         self, http_port: int, admin_port: int, no_auto: bool = False, **kwargs
     ):
         super().__init__(
-            "Faber Agent",
+            "Faber.Agent",
```

Comment:
Not sure why it's renaming the demo agents?

Reply:
When I was outputting the data from Kibana to Excel I had to use space delimiters and this was messing up the formatting.  (csv export wasn't available on the eval version of Kibana)

Reply:
Mainly wondering if that would require changes to documentation. @swcurran ?

Reply:
We can change it back if it's a big deal ...